api/src/main/resources/messages.properties

```diff
@@ -447,6 +447,7 @@ coreapps.search.identifier=Identifier
 coreapps.search.name=Name
 coreapps.search.info=Showing _START_ to _END_ of _TOTAL_ entries
 coreapps.search.label.recent=Recent
+coreapps.search.label.onlyInMpi=Only in MPI
```

Comment:
I'm not sure if 'only in MPI is good label for end-user

Reply:
Maybe something like: 'remote', 'external', 'not imported'

Reply:
I think 'not imported' will be the best one. However, I will send an email to Nathaelf about this.

Reply:
Right, I'm looking forward to his response.